[ES] Update `Main Page` and `Article styling criteria` (#8501)
ppy/osu-wiki: wiki/Main_Page/es.md

---
layout: main_page
---

<!-- Do not add any empty lines inside this div. -->

<div class="wiki-main-page__blurb">
Bienvenido a la osu! wiki, la base de conocimientos de código abierto que contiene una amplia gama de información relacionada con osu!, un juego de ritmo gratuito.
</div>

<div class="wiki-main-page__panels">
<div class="wiki-main-page-panel wiki-main-page-panel--full">

# Para empezar

> Artículos y guías importantes que te ayudarán a sumergirte en osu!. En caso de que algo salga mal o parezca incierto, asegúrese de visitar el [foro de ayuda](https://osu.ppy.sh/forum/5).

Reviewer: I think the usage of `asegúrese` is a bit odd. This is because `que te ayudarán` references the user directly while `asegúrese` references anyone.

```suggestion
Artículos y guías importantes que te ayudarán a sumergirte en osu!. En caso de que algo salga mal o parezca incierto, visita el [foro de ayuda](https://osu.ppy.sh/forum/5).
```

> 

Reviewer: While we are here, I would change the `Juego` title to `Jugabilidad` as its a better translation for `Gameplay`

[Modos de juego](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)

[Beatmap](/wiki/Beatmap) • [Dificultad](/wiki/Beatmap/Difficulty) • [Objetos](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Puntuación](/wiki/Gameplay/Score) • [Repetición](/wiki/Gameplay/Replay) • [Medallas](/wiki/Medals) • [Multi](/wiki/Client/Interface/Multiplayer)

</div>
<div class="wiki-main-page-panel">

# [Editor de beatmaps](/wiki/Client/Beatmap_editor)

Todos los beatmaps de osu! son creados por la imaginación desbordante de su comunidad dinámica. ¿Siempre quisiste compartir tu música favorita con los demás, pero nunca supiste cómo? Familiarícese con el editor integrado y las oportunidades que ofrece.

Secciones: [Composición](/wiki/Client/Beatmap_editor/Compose) • [Diseño](/wiki/Client/Beatmap_editor/Design) • [Tiempo](/wiki/Client/Beatmap_editor/Timing) • [Configuración de la canción](/wiki/Client/Beatmap_editor/Song_Setup)

> 

Reviewer: I would change `Divisor de golpes` and `Distancia de golpes` for `Division de ritmo` and `Distancia de ritmo` in the `Componentes` section here. I believe is the proper terminology we use [nowadays too](https://github.com/ppy/osu-wiki/blob/43c63c9cecca99d29e5e1d7dfe66daabceb59f2a/wiki/Client/Beatmap_editor/Beat_Snap_Divisor/es.md?plain=1#L5)

Reviewer: Should be *Divisor, my bad.

# Envío y clasificación de Beatmap

La clasificación es el proceso de obtener una tabla de clasificación para un beatmap, lo que lo hace más popular y reconocido. Descubra lo que se necesita para recorrer este camino y cómo aumentar sus posibilidades de éxito a través de las revisiones de su beatmap, así como los procedimientos de control de calidad y el proceso de clasificación.

> [Envío](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimiento de clasificación](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Community/Mappers_Guild) • [Proyecto Amado](/wiki/Community/Project_Loved)

Reviewer: While we are here I would rename:
```suggestion
[Envío](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimiento de clasificación](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Community/Mappers_Guild) • [Proyect Loved](/wiki/Community/Project_Loved)
```

Given it is a "nombre propio" considering it is capitalized.

Lea los procedimientos y normas que garantizan la seguridad de osu! y su comunidad, desde las licencias de música y las pautas de los medios hasta las reglas del chat y del foro.

> [Reglas](/wiki/Rules) • [Código de conducta del colaborador](/wiki/Contributor_Code_of_Conduct) • [Código de conducta para modding y mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)

Reviewer: ```suggestion
[Reglas](/wiki/Rules) • [Código de conducta para colaboradores](/wiki/Contributor_Code_of_Conduct) • [Código de conducta para modding y mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)
```

# [Personas](/wiki/People)

> osu! no hubiera sido posible sin la ayuda de muchos usuarios con el desarrollo, el mantenimiento y la gestión de la comunidad: su esfuerzo, tiempo y dedicación hacen que el juego viva y prospere.

Reviewer: I dont think we should add `el/la` to this cases.

> [El equipo](/wiki/People/The_Team): [Desarrolladores](/wiki/People/The_Team/Developers) • [Equipo de Moderación Global](/wiki/People/The_Team/Global_Moderation_Team) • [Equipo de Soporte](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Equipo del Proyecto Amado](/wiki/People/The_Team/Project_Loved_Team) • [Beatmap Spotlight Curators](/wiki/People/The_Team/Beatmap_Spotlight_Curators)

Reviewer: Same thing with project loved here.

I think having `Equipo` capitalized is correct here, similarly to what was pointed out earlier with project loved.

Interactúa con osu! a través de otras aplicaciones y medios, o contribuir a su desarrollo.

[API](/wiki/osu!api) • [Cuenta bot](/wiki/Bot_account) • [Directrices de identidad de marca](/wiki/Brand_identity_guidelines) • [Servidor de Discord osu!dev](/wiki/Community/osu!dev_Discord_server)

</div>
<div class="wiki-main-page-panel">

# Wiki y metapáginas

Así como casi todo por aquí, la osu! wiki está escrita y mantenida por voluntarios. Sugiera una mejora, informe un problema o incluso escriba su propio artículo en el [repositorio osu-wiki en GitHub](https://github.com/ppy/osu-wiki).

[Historia de osu!](/wiki/History_of_osu!) • [Historia de la osu! wiki](/wiki/History_of_osu!/osu!_wiki) • [Cronología de mapping y modding](/wiki/History_of_osu!/Mapping_and_Modding_Timeline) • [April Fools](/wiki/History_of_osu!/April_Fools)

> [Mapa del sitio](/wiki/Sitemap) • [Guía de contribución](/wiki/osu!_wiki/Contribution_guide) • [Criterios de estilo del artículo](/wiki/Article_styling_criteria) • [Criterios de estilo de las noticias](/wiki/News_styling_criteria)

Reviewer: Change this one to `Criterios de estilo para artículos`

Interactúa con osu! a través de otras aplicaciones y medios, o contribuir a su desarrollo.

[API](/wiki/osu!api) • [Cuenta bot](/wiki/Bot_account) • [Directrices de identidad de marca](/wiki/Brand_identity_guidelines) • [Servidor de Discord osu!dev](/wiki/Community/osu!dev_Discord_server)

</div>
<div class="wiki-main-page-panel">

# Wiki y metapáginas

> Así como casi todo por aquí, la osu! wiki está escrita y mantenido por voluntarios. SugSugiera una mejora, informe un problema o incluso escriba su propio artículo en el [repositorio osu-wiki en GitHub](https://github.com/ppy/osu-wiki).

Reviewer: ```suggestion
Así como casi todo por aquí, la osu! wiki está escrita y mantenida por voluntarios. Sugiera una mejora, informe un problema o incluso escriba su propio artículo en el [repositorio osu-wiki en GitHub](https://github.com/ppy/osu-wiki).
```
